db/seeds.rb

```diff
@@ -87,16 +87,82 @@
   mike.passive_friendships.create(active_friend: User.find_by(username: "user#{n+9}"))
 end
 
-2.times do |n|
+4.times do |n|
```

Comment:
You can use `FactoryBot` to create lists and use FactoryBot to create traits, that will allow you to have this seed more smaller/break-into-factories.

Reply:
Done! :) Though for lines that would be shorter if FactoryBot wasn't used, I just kept the same.